lib/board.rb

```diff
@@ -0,0 +1,61 @@
+require 'pry'
+
+class Board
+  attr_accessor :cells, :board
+
+  def initialize
+    # @board= @cells
+    @cells= Array.new(9, " ")
+  end
+
+  def reset!
+    @cells= Array.new(9, " ")
+  end
+
+  def display
+    puts "\n"
+    puts "\n"
+    puts " #{cells[0]} | #{cells[1]} | #{cells[2]} "
+    puts "-----------"
+    puts " #{cells[3]} | #{cells[4]} | #{cells[5]} "
+    puts "-----------"
+    puts " #{cells[6]} | #{cells[7]} | #{cells[8]} "
+    puts "\n"
+    puts "\n"
+  end
+
+  def position(input)
+    @cells[input.to_i - 1 ]
+  end
+
+  def full?
+     @cells.include?(" " ||"") ? false : true
+  end
+
+  def turn_count
+    count = 0
+    cells.each do |cell|
```

Comment:
Use the highest level and most expressive iterator possible to solve a problem. [`count`](http://ruby-doc.org/core-2.2.0/Array.html#method-i-count) on the cells is perfect and the rest of this method is not needed. Entire body could read:

```ruby
def turn_count
  cells.count { |cell| cell == "X" || cell == "O" }
end
```